src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -716,7 +730,15 @@ def status(self):
         }
         if not SubvolumeOpSm.is_complete_state(state) and subvolume_type == SubvolumeTypes.TYPE_CLONE:
             subvolume_status["source"] = self._get_clone_source()
-            subvolume_status["progress"] = self._get_clone_progress()
+            try:
+                subvolume_status["progress"] = self._get_clone_progress()
+            except VolumeException:
+                pass
+        if SubvolumeOpSm.is_failed_state(state) and subvolume_type == SubvolumeTypes.TYPE_CLONE:
```

Comment:
@ajarr @vshankar Should we treat `canceled` state as failure? If so it  might need a failure status update while transitioning from `pending` to `canceled`.

Reply:
The code is updated to display the failure status when the clone is in `failed` and `canceled` state. When in clone is cancelled, failure error `EINTR` is shown. I hope that is fine.

Reply:
For the user canceled case, lets show "User Interrupted" in the error string (relying on `strerror(EINTR)` could be misleading)

Reply:
Done. I have chosen "user interrupted clone operation"